librz/core/linux_heap_jemalloc.c

```diff
@@ -124,14 +124,14 @@ static void GH(jemalloc_get_chunks)(RzCore *core, const char *input) {
 	RzConsPrintablePalette *pal = &rz_cons_singleton()->context->pal;
 
 	if (!GH(rz_resolve_jemalloc)(core, "je_chunksize", &cnksz)) {
-		eprintf("Fail at read symbol je_chunksize\n");
+		rz_cons_printf("Fail at read symbol je_chunksize\n");
```

Comment:
RZ_LOG_ERROR

Reply:
I thought the same at first. However, I favored to leave the LOG_ERROR for functions that doesn't exit with exit(0).

Reply:
no need, when you throw an error it is always RZ_LOG_ERROR